syntax = "proto3";

package lynx.pb;

enum RemoteWordType {
  NOUN = 0;
  ADJECTIVE = 1;
  VERB = 2;
  ADVERB = 3;
}

message RemoteWordImage {
  uint64 id = 1;
  string url = 2;
  int32 width = 3;
  int32 height = 4;
}

message RemoteWord {
  uint64 id = 1;
  string name = 2;
  uint64 index = 3;
  RemoteWordType type = 4;
  RemoteWordImage image = 5;
}
